packages/cli/jest.config.ts

```diff
@@ -1,18 +0,0 @@
-/* eslint-disable */
-export default {
-  displayName: 'cli',
-  preset: '../../jest.preset.js',
-  globals: {},
-  transform: {
-    '^.+\\.[tj]s$': [
-      'ts-jest',
-      {
-        tsconfig: '<rootDir>/tsconfig.spec.json',
-      },
-    ],
-  },
-  transformIgnorePatterns: ['/node_modules/(?!markdown-table/.*)'],
-  moduleFileExtensions: ['ts', 'js', 'html'],
-  coverageDirectory: '../../coverage/packages/cli',
-  testTimeout: 40_000,
```

Comment:
I noticed the `testTimeout` was not increased to 40 seconds for `vitest`. Is it not needed anymore?

Reply:
The configuration you are talking about was never used. It was incorrect and would throw a warning. So yeah, I dont want to have an additional timeout :D